app/services/step_code/compliance/propose_step/base.rb

```diff
@@ -28,10 +29,6 @@ def requirements_met?
     checkers.all? { |c| self.send(c).requirements_met? }
   end
 
-  def min_required_step
```

Comment:
Why is this done with min_required_step but not max step?

Reply:
min step is now passed in as a param on initialization since it's no longer static (e.g. it depends on the permit type required step currently being tested)